dolphin/db/sqlalchemy/api.py

```diff
@@ -303,25 +322,67 @@ def volume_get_all(context, marker=None, limit=None, sort_keys=None,
     return NotImplemented
 
 
-def pool_create(context, values):
+def pool_create(context, values, session=None):
     """Create a pool from the values dictionary."""
     return NotImplemented
 
 
-def pool_update(context, pool_id, values):
+def pool_update(context, pools):
```

Comment:
Pool update is for a single pool , right?

Reply:
No. Its a list of pools